lib/raster/json_color_out.c

```diff
@@ -47,7 +47,7 @@ static void close_file(FILE *fp)
    \param[out] s pointer to store the calculated saturation
    \param[out] v pointer to store the calculated value
  */
-static void rgb_to_hsv(int r, int g, int b, float *h, float *s, float *v)
+void Rast_rgb_to_hsv(int r, int g, int b, float *h, float *s, float *v)
```

Comment:
I am thinking a better place for this general function would be perhaps in lib/gis/color_str.c